infra/resources/_modules/apim/send.tf

```diff
@@ -0,0 +1,60 @@
+resource "azurerm_api_management_product" "apim_itn_product_send_aar" {
+  product_id   = "io-send-aar-api"
+  display_name = "IO SEND AAR API"
+  description  = "Product for IO SEND AAR API"
+
+  api_management_name = data.azurerm_api_management.apim_itn_api.name
+  resource_group_name = data.azurerm_api_management.apim_itn_api.resource_group_name
+
+  approval_required = false
+  published         = true
+}
+
+resource "azurerm_api_management_product_policy" "apim_itn_product_send_policy" {
+  product_id          = azurerm_api_management_product.apim_itn_product_send_aar.product_id
+  api_management_name = data.azurerm_api_management.apim_itn_api.name
+  resource_group_name = data.azurerm_api_management.apim_itn_api.resource_group_name
+
+  xml_content = file("../_modules/apim/product/policy.xml")
```

Comment:
Maybe [xml_link](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/api_management_product_policy#xml_link-1) is better here